integrations/fullstory/manifest.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "FullStory",
+  "version": "1.0.0",
+  "overview": "Send feature flag evaluation data from the LaunchDarkly JavaScript SDK to FullStory as page properties.",
```

Comment:
Ugh it seems like this is 3 chars too long.

Reply:
@ldhenry removed the word `evaluation`. The docs still cover the exact functionality so I think it's an ok tradeoff.